packages/astro/src/vite-plugin-markdown/index.ts

```diff
@@ -161,12 +161,16 @@ export default function markdown({ config }: AstroPluginOptions): Plugin {
 				const { layout = '', components = '', setup = '', ...content } = frontmatter;
 				content.astro = metadata;
 				const prelude = `---
-import { slug as $$slug } from '@astrojs/markdown-remark/ssr-utils';
```

Comment:
Remove direct import from `@astrojs/markdown-remark` sub-dependency. Since this isn't explicitly installed, `astro build` will fail from our monorepo